data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg

```diff
@@ -207,12 +215,12 @@
 
         [message]
             speaker=Malin Keshar
-            message= _ "Since the very founding of Wesnoth you have hounded humans at every turn. You have raided my home at every opportunity. You have trampled our fields, slaughtered our livestock. You have brought war to a place of peace, and death to a place of life. You have killed without remorse, and despoiled that which was once beautiful. You have murdered my father and caused endless trouble to me. Now I shall have my revenge on you. Expect no mercy from me, because you have long since lost the right to it. I will crush you into the dust. The attacks on Parthyn will stop here and now. When I bring word of your demise the town will cheer my triumphant return."
+            message= _ "Since the very founding of Wesnoth you have hounded humans at every turn. You have trampled our fields, slaughtered our livestock, and burned our houses. You have brought war to a place of peace and death to a place of life. You are nothing but remorseless killers and savage beasts. Your kind deserves no mercy and shall receive none. I will crush you into the dust and make you into my slaves for all eternity."
```

Comment:
Given the events in TRoW, this could be "Even before the very founding of Wesnoth"